.github/workflows/robustness-template.yaml

```diff
@@ -72,9 +72,7 @@ jobs:
               EXPECT_DEBUG=true GO_TEST_FLAGS=${GO_TEST_FLAGS} RESULTS_DIR=/tmp/results make test-robustness-release-3.4
               ;;
             main)
-              make gofail-enable
-              make build
-              EXPECT_DEBUG=true GO_TEST_FLAGS=${GO_TEST_FLAGS} RESULTS_DIR=/tmp/results make test-robustness
+              EXPECT_DEBUG=true GO_TEST_FLAGS=${GO_TEST_FLAGS} RESULTS_DIR=/tmp/results make test-robustness-release-3.6
```

Comment:
Didn't notice it earlier.

Should the target be called `test-robustness-release-3.6`, if we are testing `main` branch?

Reply:
change it to `test-robustness-main`?

Reply:
SG